src/game.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+/* eslint-disable no-console */
+
+const { getRandomNumber } = require('./generateRandomNumber');
+const { calculateAnimals } = require('./calculateAnimals');
+const { terminal } = require('./terminal');
+const dotenv = require('dotenv');
+const {
+  UNIQUE_NUMBER_ERROR,
+  NUMBER_LENGTH_ERROR,
+  TYPEOF_ERROR,
+  YOU_WIN,
+  ANSWER_ERROR,
+  SEE_YOU,
+  ENTER_THE_NUMBER,
+  WELCOME,
+} = require('./constants');
+
+dotenv.config();
+
+const {
+  START,
+  CONTINUE,
+  EXIT,
+} = process.env;
+
+const randomNumber = getRandomNumber();
+
+console.log(WELCOME);
+
+const playGame = () => {
+  setTimeout(() => {
+    terminal.question(START, (answer) => {
+      if (answer === CONTINUE) {
+        const game = () => {
+          terminal.question(ENTER_THE_NUMBER, (userNumber) => {
+            if (userNumber.length !== 4) {
```

Comment:
if-else hell 🥲 

Maybe we can do it cleaner and without such a count of nesting?